bower.json

```diff
@@ -6,6 +6,7 @@
   ],
   "dependencies": {
     "o-brand": "^3.2.5",
+    "o-buttons": "^6.2.0",
```

Comment:
This is not breaking as it's the only o-buttons major compatible with this o-forms major (there's no chance of a conflict)